release-tools/build.make

```diff
@@ -74,7 +74,7 @@ endif
 # separated by semicolon. An empty variable or empty entry (= just a
 # semicolon) builds for the default platform of the current Go
 # toolchain.
-BUILD_PLATFORMS =
+BUILD_PLATFORMS = linux amd64
```

Comment:
Might not need this if CI sets this up to build multi-arch images. cc: @hairyhum

Reply:
Can we move that to `Makefile` instead? `build.make` should not be changed from the base subtree.

Reply:
Sure, makes sense